### PR TITLE
ComboBox: Fix scrolling issue

### LINKED ISSUE #2090: Combobox: Can't select some options in controlled Combobox
### Bug Report

- __Package version(s)__: latest master
- __Browser and OS versions__: chrome

#### Priorities and help requested (not applicable if asking question):

Requested priority: (High) 

#### Describe the issue:

Can't select some options in controlled combobox.

### If applicable, please provide a codepen repro:

1. Pull and build latest master
2. Go to combobox page, find the "Basic controlled example"
image
3. Try use mouse to select any one of "Arial Black", "Time New Roman", or "Comic Sans MS"

Actual: The focus jumps to the previously selected option, the option I clicked is not selected.
Expected: The combobox close and the input shows the item I just clicked

Note: The issue is not stable. (1 out of 5 reproduced after refreshing the page) Try refreshing and reproduce for several times.
combo-box

Look at the third time refreshing in the gif

### PR DESCRIPTION
**Pull request by jspurlin**
…ect some options in controlled Combobox #2090)

#### Pull request checklist

- [X] Addresses an existing issue: Fixes #2090 
- [X] Include a change request file using `$ npm run change`

#### Description of changes

When determining when to scroll the selected item in to view, the code was too aggressive and would re-scrollIntoView/focus the selected option when the menu was already open.

The fix is to make sure we don't attempt to scrollIntoView/focus when the menu is already open (e.g. the previous open state is true)

#### Focus areas to test

Verified that when a scrollbar is present we can still select any option, especially options that cause the previously selected option to be out of view

### CODE REVIEW

File: common/changes/@uifabric/example-app-base/jspurlin-ComboBoxScrollIntoViewFix_--2090-_2017-06-29-13-27.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
```
Comment (jair-cazarin) [inline common/changes/@uifabric/example-app-base/jspurlin-ComboBoxScrollIntoViewFix_--2090-_2017-06-29-13-27.json]: what's with these extra change files?

Comment (jspurlin) [inline common/changes/@uifabric/example-app-base/jspurlin-ComboBoxScrollIntoViewFix_--2090-_2017-06-29-13-27.json]: <shrug> That's what "rush change" spit out even though I just pressed ENTER to advance past the info for the other packages (which is what it says to do)

Comment (jair-cazarin) [inline common/changes/@uifabric/example-app-base/jspurlin-ComboBoxScrollIntoViewFix_--2090-_2017-06-29-13-27.json]: Did you rush change against the remote branch?

Comment (jair-cazarin) [inline common/changes/@uifabric/example-app-base/jspurlin-ComboBoxScrollIntoViewFix_--2090-_2017-06-29-13-27.json]: -b BRANCH, --target-branch BRANCH
                        If this parameter is specified, compare current 
                        branch with the target branch to get changes. If this 
                        parameter is not specified, the current branch is 
                        compared against the "master" branch.

**Comment by jspurlin** [comment]
@dzearing ... is Travis having issues...? (it's not reporting status)

**Pull request merged**